abi/develop/HolographOperator.json

```diff
@@ -7,20 +7,14 @@
   {
     "anonymous": false,
     "inputs": [
-      {
-        "indexed": false,
-        "internalType": "bytes32",
-        "name": "jobHash",
-        "type": "bytes32"
-      },
       {
         "indexed": false,
         "internalType": "bytes",
-        "name": "payload",
+        "name": "_payload",
```

Comment:
Are these changes intentional

Reply:
Yep, this is `feature/operator-pod-logic` branch pollution. When transitioning to new branch structuring, we mixed some code from that branch. This is me cleaning it up.

Reply:
ah okay cool